flexmeasures/data/schemas/sensors.py

```diff
@@ -28,7 +29,10 @@ class Meta:
     name = ma.auto_field(required=True)
     unit = ma.auto_field(required=True)
     timezone = ma.auto_field()
-    event_resolution = fields.TimeDelta(required=True, precision="minutes")
+    event_resolution = fields.TimeDelta(precision="minutes")
+    resolution = NewDurationField(
```

Comment:
NewDurationField has been removed. `event_resolution` is now a DurationField.
DurationField has added validation for a minimum of 1 minute.
To make sure the CLI can still use the int minutes as `event_resolution` a check has been added to the CLI and a deprication message that this will be changed to a `isodate.duration` string in the next version.